src/components/enterprise-user-subsidy/tests/AutoActivateLicense.test.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable react/jsx-no-constructed-context-values */
```

Comment:
Similar question about being consistent on how we're handling this ESLint rule (in some test files, you're using useMemo, but in others like here, you're not).

Reply:
If we plan on not enforcing this rule in tests, should we update this repos ESLint config to ignore this rule for `*.test.js(x)` files to not have to ignore the rule all over the place?